src/menu_items.c

```diff
@@ -3400,15 +3400,15 @@ void clear_menu_textures(void) {
  * @return void*
  */
 void* segmented_to_virtual_dupe(const void* addr) {
-    size_t segment = (uintptr_t) addr >> 24;
-    size_t offset = (uintptr_t) addr & 0x00FFFFFF;
+    ssize_t segment = (uintptr_t) addr >> 24;
+    ssize_t offset = (uintptr_t) addr & 0x00FFFFFF;
 
     return (void*) ((gSegmentTable[segment] + offset) + 0x80000000);
 }
 
 void* segmented_to_virtual_dupe_2(const void* addr) {
-    size_t segment = (uintptr_t) addr >> 24;
-    size_t offset = (uintptr_t) addr & 0x00FFFFFF;
+    ssize_t segment = (uintptr_t) addr >> 24;
+    ssize_t offset = (uintptr_t) addr & 0x00FFFFFF;
```

Comment:
Please fix

Reply:
I dont think it needs to be ssize_t, the value is never going to be negative

Reply:
Ah nvm im stupid

Reply:
yes?

Reply:
![image](https://github.com/user-attachments/assets/83bf72ef-a063-4a02-b7fa-d86eb6b43d9f)

Reply:
no idea how that happened